docs/releases/creating-a-release.md

```diff
@@ -6,27 +6,44 @@ Along with the NPM package, we also update the documentation hosted on [storyboo
 This repository follows a green-trunk development model. All development happens on the `main` branch. Releasing a new version of the package involves three distinct steps:
 
 1. [Create a git branch to release the package](#step-1-creating-a-release-branch). This step is mostly automated via GitHub workflows.
-1. [Prepare the branch for release](#step-2-prepare-for-release). This step involves API approvals, string translations, bug bashes etc and tends to last 1-2 weeks.
-1. [Release the package to NPM for release branch](#step-3-publish-to-npm). This step is mostly automated via GitHub workflows.
+2. [Prepare the branch for release](#step-2-prepare-for-release). This step involves API approvals, string translations, bug bashes etc and tends to last 1-2 weeks.
+3. [Release the package to NPM for release branch](#step-3-publish-to-npm). This step is mostly automated via GitHub workflows.
```

Comment:
in markdown we can actually just leave it as `1.` for numbered lists. Its probably preferred cause it will automatically increment the count for us. So if we ever need to hypothetically add something in the middle we wouldn't need to edit the whole

Reply:
Oh that makes sense! I'll fix these.